scripts/commands/terraform.sh

```diff
@@ -30,5 +30,5 @@ terraform() {
         exit 1
     fi
 
-    printf '{"content_base64":"%s"}' "$(printf '%s' "${content}" | base64)"
+    printf '{"content_base64":"%s"}' "$(printf '%s' "${content}" | base64 | tr -d \\n)"
```

Comment:
I would prefer to use `base64 -w0` instead of an additional pipe/subprocess.

But there is no `-w` parameter on MacOS (but `-w0` is default). I guess its fine for now.

Reply:
Right. I was about to use -w but ran a last minute check on local and got invalid flag error.